src/test/scala/cognite/spark/v1/AssetsRelationTest.scala

```diff
@@ -145,9 +146,13 @@ class AssetsRelationTest extends FlatSpec with Matchers with SparkTest {
     val assetsCreated = getNumberOfRowsCreated(metricsPrefix, "assets")
     assert(assetsCreated == 1)
 
-    retryWhile[Array[Row]](
+    val Array(createdAsset) = retryWhile[Array[Row]](
```

Comment:
Pro tip: You can use [`loneElement`](http://doc.scalatest.org/3.0.1-2.12/org/scalatest/LoneElement.html) to get better error messages when you expect a collection to contain a single element. I don't consider that a blocker for this PR though.